src/commonReducers/clients/clientSlice.js

```diff
@@ -0,0 +1,46 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import sources from 'api/sources';
+
+
+export const fetchClientsById = createAsyncThunk(
+  'clients/fetchClientsStatus',
+  async (userID) => {
+    const response = await sources.clientList.get(userID);
+    return response.json();
+  },
+);
+
+export const addClient = createAsyncThunk(
+  'clients/getClients',
+  async (data) => {
+    const response = await sources.clientList.post(data);
+    return response;
+  },
+);
+
+export const clientsSlice = createSlice({
+  name: 'clients',
```

Comment:
this is really nitpicky and opinionated - but usually I would call a reducer something like `clientReducer` or just `client`. What you're defining here is a 'slice' of your app store. So would be the 'client' slice.

So accessing a list of all clients would become something like `state.client.clients`